src/IssueViz/Editor/ErrorTagging/ErrorTagger.cs

```diff
@@ -45,9 +45,15 @@ protected override TagSpan<IErrorTag> CreateTagSpan(IIssueLocationTag trackedTag
         }
 
         protected override IEnumerable<IMappingTagSpan<IIssueLocationTag>> Filter(IEnumerable<IMappingTagSpan<IIssueLocationTag>> trackedTagSpans) =>
-            trackedTagSpans.Where(x => IsValidPrimaryLocation(x.Tag.Location));
+            trackedTagSpans.Where(x => IsValidPrimaryLocation(x.Tag.Location) && IsNotSuppressed(x.Tag.Location));
 
         private static bool IsValidPrimaryLocation(IAnalysisIssueLocationVisualization locViz) =>
             locViz is IAnalysisIssueVisualization && locViz.Span.IsNavigable();
+
+        private static bool IsNotSuppressed(IAnalysisIssueLocationVisualization locViz)
+        {
+            var issueViz = (IAnalysisIssueVisualization)locViz;
```

Comment:
`(IAnalysisIssueVisualization)locViz` is a hard cast, which will fail if the types don't match.
The old `IsValidPrimaryLocation` used `is IAnalysisIssueVisualization` which is a soft cast, and will return `false` if the types don't match.

The suggested new version of the code just uses `is`.